src/chocolatey/infrastructure.app/validations/SystemStateValidation.cs

```diff
@@ -84,7 +84,7 @@ private void ValidateSystemPendingReboot(ChocolateyConfiguration config, ICollec
    being ignored due to the current Chocolatey configuration.  If you
    want to halt when this occurs, then either set the global feature
    using:
-     choco feature enable -name={0}
+     choco feature enable --name={0}

```

Comment:
Same question as above

```suggestion
     choco feature enable --name=""{0}""
```

Reply:
Same answer as above 😄